bindings/python/convert.py

```diff
@@ -96,12 +103,16 @@ def convert_single(model_id: str, folder: str) -> List["CommitOperationAdd"]:
     # For tensors to be contiguous
     loaded = {k: v.contiguous() for k, v in loaded.items()}
 
-    save_file(loaded, local, metadata={"format": "pt"})
-
-    check_file_size(local, filename)
-
-    operations = [CommitOperationAdd(path_in_repo=sf_filename, path_or_fileobj=local)]
-    return operations
+    dirname = sf_filename.rsplit("/", 1)[0]
```

Comment:
```suggestion
    dirname = sf_filename.rsplit(os.path.sep, 1)[0]
```
Won't work on Windows otherwise.